returnn/torch/engine.py

```diff
@@ -669,6 +672,139 @@ def forward_with_callback(self, *, dataset: Dataset, callback: ForwardCallbackIf
 
             callback.finish()
 
+    @staticmethod
+    def delete_model(filename):
```

Comment:
This looks like basically the same logic, even almost the same code from the TF engine, copy and pasted here. Why? Why not move it to the engine base class instead, to share the same code for both?